src/components/BottomTabBarIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import FastImage, { FastImageSource } from 'react-native-fast-image';
+import { StyleSheet } from 'react-native';
+
+export interface BottomTabBarIconProps {
+  source: FastImageSource;
+}
+
+export class BottomTabBarIcon extends React.PureComponent<BottomTabBarIconProps> {
+  render() {
+    const { source } = this.props;
+    return <FastImage source={source} style={styles.image} />;
```

Comment:
consider adding a custom image component to `src/components/Image` that would just re-export FastImage

We would have additional abstraction layer - we would never use 
```
import FastImage, { FastImageSource } from 'react-native-fast-image';
```
in our other components (except `Image.tsx` - in case something happens and we need to replace FastImage with something else, we will need just single code change)

Reply:
Image.tsx:
```
export { default as Image } from 'react-native-fast-image'
```
that's it